src/tools/hidpp-list-features.cpp

```diff
@@ -69,8 +69,10 @@ static const std::map<uint16_t, const char *> HIDPP20Features = {
 	{ 0x1bc0, "Persistent remappable action" },
 	{ 0x1d4b, "Wireless device status" },
 	{ 0x1e00, "Enable hidden features" },
+	//{ 0x1eb0, "Seen on Logitech Litra Glow" },
```

Comment:
`1EB0` is also used in a number of other devices, like a [mouse from 2015](https://github.com/pwr-Solaar/Solaar/issues/208), [another mouse](https://github.com/pwr-Solaar/Solaar/issues/672), etc.
Short of knowing what it is, I don't see a reason to add it to the list.

Reply:
It's an internal generic test/development feature that's not useful to users and won't be publicly documented. Devices do different things with that feature and it's not specific to the Litra Glow.

Reply:
Isn't the documentation, that it IS a test feature a good reason to add it to the list - even only as a comment?

Reply:
I'd say that depends on the purpose of the list. It seems it mostly lists features that showed up in public documentation. But one could obviously shoot for completeness, in which case `{ 0x1eb0, "Internal test feature" }` would be an entirely reasonable entry. :smile: